pkg/bypass4netns/bypass4netns.go

```diff
@@ -800,16 +799,13 @@ func (h *Handler) StartHandle(c2cConfig *C2CConnectionHandleConfig, multinodeCon
 		notifHandler.c2cConnections = c2cConfig
 		notifHandler.multinode = multinodeConfig
 		if notifHandler.multinode.Enable {
-			notifHandler.multinode.etcdClientConfig = client.Config{
-				Endpoints:               []string{notifHandler.multinode.EtcdAddress},
-				Transport:               client.DefaultTransport,
-				HeaderTimeoutPerRequest: 2 * time.Second,
```

Comment:
(Not sure if there is an alternative to `HeaderTimeoutPerRequest` in the new API)

Reply:
I think `context.WithCancel` in each API call can replace the timeout.
https://pkg.go.dev/go.etcd.io/etcd/client/v3#section-readme